public-docker-images/README.md

```diff
@@ -0,0 +1,52 @@
+# Veneur, in Docker
+
+This folder holds the Docker resources for building [Veneur](https://github.com/stripe/veneur), and a sample systemd unit for running it. It lives in the Veneur repository, but currently clones from Github, so the state of the repository you build in is unrelated to the output.
+
+## Building
+
+The `--no-cache` argument may be necessary to fetch the latest commits from Github, unless you're building a specific version tag.
+
+```
+public-docker-images$ docker build --no-cache -t veneur:local -f Dockerfile-debian-sid .
+```
+
+For the Alpine Linux-based image:
+```
+public-docker-images$ docker build --no-cache -t veneur:local --build-arg Dockerfile-alpine .
+```
+
+For a specific tag or branch:
+```
+public-docker-images$ docker build -t veneur:local --build-arg BUILD_REF='tag-or-branch' Dockerfile-debian-sid .
```

Comment:
Similarly, here I think we want `$ $ docker build -t veneur:local --build-arg BUILD_REF='v1.7.0' -f Dockerfile-debian-sid .`

Reply:
Although even that doesn't quite work for me - I get this build failure:

```
Step 36/41 : WORKDIR /veneur/
 ---> Using cache
 ---> d8553e80ed96
Step 37/41 : EXPOSE 8126/UDP 8126/TCP 8127/TCP 8128/UDP
 ---> Using cache
 ---> b6d4526c3001
Step 38/41 : COPY --from=release-prep /etc/ca-certificates.conf /etc/ca-certificates.conf
invalid from flag value release-prep: pull access denied for release-prep, repository does not exist or may require 'docker login'
```

Reply:
This is an error -- this line should be removed now

Reply:
Re: `should be BUILD_REF='v1.7.0'` - these are instructions, not examples, so I favor using demonstrative names (in some format) rather than specific valid values when the specificity is not required. I'm not averse to adding a separate example or quickstart section, though. OK by you?

Reply:
Sorry, yeah, I agree with that. I meant the `-f` bit. As written, the example doesn't work on my machine (even if I provide the correct value for `tag-or-branch`).

Reply:
Blast. I usually try examples as I write them, don't know what I was doing here...